sky/optimizer.py

```diff
@@ -113,11 +123,25 @@ def optimize(dag: 'dag_lib.Dag',
         # node.best_resources if it is None.
         Optimizer._add_dummy_source_sink_nodes(dag)
         try:
-            unused_best_plan = Optimizer._optimize_objective(
-                dag,
-                minimize_cost=minimize == OptimizeTarget.COST,
-                blocked_resources=blocked_resources,
-                quiet=quiet)
+
+            if _is_dag_with_res_list(dag):
+                # Honor the user's choice.
+                logger.info(
+                    f'{colorama.Fore.YELLOW}Using user-specified accelerators list{colorama.Style.RESET_ALL} (will be tried in the listed order).'  # pylint: disable=line-too-long
+                )
+                resources_list = dag.tasks[0].get_resources_list()
+                logger.info([
```

Comment:
Nit: may want to print this with L129. This will make the output log use 1 line vs. 2.

Reply:
Thanks updated.